src/Datadog.Trace.ClrProfiler.Managed/Integrations/ElasticsearchNet6Integration.cs

```diff
@@ -49,6 +49,7 @@ public static object CallElasticsearch<TResponse>(object pipeline, object reques
                     .WithConcreteType(pipelineType)
                     .WithMethodGenerics(genericArgument)
                     .WithParameters(requestData)
+                    .WithNamespaceAndNameFilters(null, "Elasticsearch.Net.RequestData")
```

Comment:
There's some inconsistency in using `ClrNames.Ignore` vs `null`. Personally, I prefer `null`, but either one is fine as long as we're consistent.

```suggestion
                    .WithNamespaceAndNameFilters(ClrNames.Ignore, "Elasticsearch.Net.RequestData")
```

Reply:
Benefit of using `ClrNames.Ignore` is that it's consistent with our native checks and it has to be very intentional.
Is that okay with you?

Reply:
Yup, thanks! Not in this PR, but it might be worth looking into getting these types from the `[InterceptMethod]` or the `integrations.json` file to reduce duplicating error-prone code.

Reply:
Whoops, I did not see this comment before echoing the same thing in my comments. Please feel free to resolve my other comments.